hash/coleco.xml

```diff
@@ -3420,4 +3420,386 @@ license:CC0-1.0
 		</part>
 	</software>
 
+<!-- Aftermarket releases -->
```

Comment:
What do you mean by “Aftermarket”?  The term refers to any parts sold to be fitted to a machine/system after it’s purchased.  The archetypal example is an “aftermarket sunroof”, i.e. an opening roof fitted to a vehicle after delivery (as opposed to a “factory sunroof”, fitted to the car before delivery).  Any cartridges or peripherals that weren’t included as pack-ins with the system are “aftermarket”.

Reply:
I intend to indicate the games made available after the commercial life of the ColecoVision system. It was for this reason that I do not mix with the others. Perhaps there is a better term for this.